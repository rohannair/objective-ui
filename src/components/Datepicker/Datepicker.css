.month.month,
.calendar.calendar {
  background-color: #fff;
  color: #444;
}
