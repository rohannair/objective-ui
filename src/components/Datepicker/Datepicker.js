import DatePicker from 'react-toolbox/lib/date_picker'

import React, { PropTypes } from 'react'
import theme from './Datepicker.css'
import { newDate } from '../../utils/dates'

const ThemedDatepicker = ({
  label,
  minDate = newDate('2017-01-01'),
  onChange,
  value
}) => (
  <DatePicker
    theme={theme}
    label={ label }
    minDate={ minDate || min_datetime }
    onChange={ onChange }
    value={ value }
  />
)

export default ThemedDatepicker
