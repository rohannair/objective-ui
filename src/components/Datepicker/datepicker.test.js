import React from 'react'

import Datepicker from './index.js'

describe('Test suite for Datepicker component', () => {
  it('Datepicker should exist', () => {
    let wrapper = shallow(<Datepicker />)
    expect(wrapper).toMatchSnapshot()
  })
})

