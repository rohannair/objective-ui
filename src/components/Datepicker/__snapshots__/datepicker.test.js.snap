exports[`Test suite for Datepicker component Datepicker should exist 1`] = `
<ThemedDatePicker
  active={false}
  composeTheme="deeply"
  locale="en"
  minDate={2015-11-08T05:00:00.000Z}
  sundayFirstDayOfWeek={false}
  theme={
    Object {
      "process": [Function],
    }
  } />
`;
